### ISSUE TITLE
 KEB endpoints exposed and secured with Istio CR

### ISSUE THREAD

**Issue by strekm** [kyma-project/control-plane#1732]
**Description**

Expose KEB endpoints under new virtual host using Istio VS, RA and AP CRs. Both configurations should work in parallel and should not interfere with each other. Based on [[POC] Expose and secure httpbin endpoints with Istio](https://github.tools.sap/kyma/backlog/issues/2528), relevant section **Traffic from oathkeeper - external JWT provider for istio-httpbin** expose following endpoints with Istio CRs:
- KEB/info/runtimes, scope cld:read, issuer ORY Hydra
- KEB/oath, scope broker:write, issuer ORY Hydra
- KEB/runtimes, scope: runtimeOperator, issuer Dex (this might change)
- KEB/orchestrations, scope: orchestrationsAdmin, issuer Dex (this might change)
- KEB/upgrade, scope: orchestrationsAdmin, issuer Dex (this might change)

For exact configuration refer to [KEB oathkeeper rule yaml](https://github.com/kyma-project/control-plane/blob/main/resources/kcp/charts/kyma-environment-broker/templates/oathkeeper-rule.yaml)

Changes should be rolled out to dev and tested, then to stage and finally to prod. Rollout to prod only after feature sits on stage for few days. This means separate PR needs to be create for each env.

**Note**

Virtual host yet needs to be determined. 
Usage of Dex needs to be determined.

**Reasons**

Dropping dependency to api-gateway and ORY oathkeeper

**Attachments**

https://github.com/kyma-project/control-plane/issues/1540
https://github.com/kyma-project/control-plane/blob/main/resources/kcp/charts/kyma-environment-broker/templates/oathkeeper-rule.yaml

**Comment by strekm** [comment]
changes rolled out to prod
